src/packet-saprfc.c

```diff
@@ -485,8 +498,51 @@ dissect_saprfc_tables_compressed(tvbuff_t *tvb, packet_info *pinfo, proto_tree *
 			tvb_set_child_real_data_tvbuff(tvb, next_tvb);
 			add_new_data_source(pinfo, next_tvb, "Uncompressed Table Data");
 
+			structure = proto_tree_add_item(tree, hf_saprfc_table_structure, structure_tvb, 4, structure_length, ENC_NA);
+			structure_tree = proto_item_add_subtree(structure, ett_saprfc);
+
+			/* Parse row structure */
+			field_count = tvb_get_guint8(structure_tvb, structure_offset); structure_offset+=1;
```

Comment:
Do you think it would be possible to check the field count for an upper limit? Is there a minimum length per field that we can use to check if we've enough payload remaining?

Reply:
Several checks were added. I hope such checks are what you meant with this?

Reply:
Yes, that's the type of checks I was referring to. Looks good!